core/src/mender-client.c

```diff
@@ -344,10 +342,9 @@ mender_client_init(mender_client_config_t *config, mender_client_callbacks_t *ca
         goto END;
     }
     mender_api_config_t mender_api_config = {
-        .artifact_name = mender_client_config.artifact_name,
-        .device_type   = mender_client_config.device_type,
-        .host          = mender_client_config.host,
-        .tenant_token  = mender_client_config.tenant_token,
+        .device_type  = mender_client_config.device_type,
+        .host         = mender_client_config.host,
+        .tenant_token = mender_client_config.tenant_token,
```

Comment:
These are not very fortunate formatting rules. Or rather, the enforced application of them...